content/en/docs/reference/command-line-tools-reference/feature-gates/disable-node-kube-proxy-version.md

```diff
@@ -6,12 +6,8 @@ _build:
   render: false
 
 stages:
-  - stage: alpha
+  - stage: deprecated
     defaultValue: false
     fromVersion: "1.29"
-    toVersion: "1.30"
-  - stage: beta
-    defaultValue: true
-    fromVersion: "1.31"
```

Comment:
```suggestion
  - stage: alpha
    defaultValue: false
    fromVersion: "1.29"
    toVersion: "1.30"
  - stage: beta
    defaultValue: true
    fromVersion: "1.31.0"
    toVersion: "1.31.0"
  - stage: alpha
    defaultValue: falsse
    fromVersion: "1.31.1"
```

Reply:
It's not the way I recommend (I'd have called the gate `LegacyNodeKubeProxyVersion` and leave it enabled by default before eventually changing it to off by default) - but we are where we are, and shouldn't edit history.

Reply:
I guess if we want to, we _could_ change the feature gate name. Shame it reached beta before we - K8s - noticed the snags.

Reply:
I'd ... do a lot of things differently if I could, but I think changing the gate name now is worse

I didn't know fromVersion / toVersion worked with patch-level versions

Reply:
They're opaque strings, nothing parses them.

Reply:
> I didn't know fromVersion / toVersion worked with patch-level versions

They should support strings in X.Y and X.Y.Z formats. If versions are not in these formats, the website build fails when rendering the feature gate table (at least, that’s how I implemented the metadata for feature gate files and used them to render the table and feature state).

Reply:
alpha->beta->alpha?

and,

false->true->false?

why?

Reply:
We're recording what has happened @tengqm; see https://github.com/kubernetes/kubernetes/pull/126720 and https://github.com/kubernetes/enhancements/issues/4004#issuecomment-2291482423 for more details.

Reply:
ok, tried recording the specific evolution at patch version level